week-8/Homework/mandatory/3-dog-photo-gallery/script.js

```diff
@@ -0,0 +1,14 @@
+document.getElementById("button").addEventListener("click", () => {
+  let makeImage = document.getElementById("dogImages");
+
+  fetch(`https://dog.ceo/api/breeds/image/random`)
+    .then(function (data) {
+      return data.json();
+    })
+    .then(function (newLink) {
+      makeImage.src = newLink.message;
+    })
+    .catch((error) => {
+      console.log(error);
```

Comment:
Maybe here you could also think about displaying an error message in case something goes wrong